documentation/modules/exploit/multi/misc/apache_activemq_rce_cve_2023_46604.md

```diff
@@ -0,0 +1,287 @@
+## Vulnerable Application
+This module exploits a deserialization vulnerability in the OpenWire transport unmarshaller in Apache ActiveMQ.
+Affected versions include 5.18.0 through to 5.18.2, 5.17.0 through to 5.17.5, 5.16.0 through to 5.16.6, and all
+versions before 5.15.16.
+
+For a full technical analysis of the vulnerability read the
+[Rapid7 AttackerKB Analysis](https://attackerkb.com/topics/IHsgZDE3tS/cve-2023-46604/rapid7-analysis).
+
+## Testing
+
+### Linux
+* The official [Getting Started](https://activemq.apache.org/getting-started) documentation has a full walkthrough.
+* You will need to install Java if you have not already done so.
+* Download a vulnerable version of ActiveMQ, e.g. [apache-activemq-5.18.3-bin.tar.gz](https://www.apache.org/dyn/closer.cgi?filename=/activemq/5.18.3/apache-activemq-5.18.3-bin.tar.gz&action=download)
```

Comment:
```suggestion
* Download a vulnerable version of ActiveMQ, e.g. [apache-activemq-5.18.2-bin.tar.gz](https://www.apache.org/dyn/closer.cgi?filename=/activemq/5.18.2/apache-activemq-5.18.2-bin.tar.gz&action=download)
```

I figure you mean the vulnerable version- it does not look like 5.18.3 is vulnerable?